oci-seccomp-bpf-hook.go

```diff
@@ -317,24 +332,40 @@ func sendSIGINT() error {
 }
 
 // generate the seccomp profile from the syscalls provided
-func generateProfile(c map[string]int, fileName string) error {
+func generateProfile(calls map[string]int, fileName string, inputProfilePath string) error {
 	s := types.Seccomp{}
+	inputProfile := types.Seccomp{}
+
+	if inputProfilePath != "" {
+		inputFile, err := ioutil.ReadFile(inputProfilePath)
```

Comment:
NIT: sed 's/inputFile/input/g'
inputFile makes me think of a file path, 
Above you pass in inputFile, and then in the function, you call it inputProfilePath..

I would prefer that you just always call it the same thing inputFile, then the content can be input, or inputData